src/VirtualClient/VirtualClient.Actions/PostgreSQL/balanced.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e
+
+if [ $# -ge 1 ]; then
```

Comment:
Does this have anything that's ubuntu specific? If not let's not have ubuntu in the path. Or if at least it works in both deb and ubuntu, name it deb because it's super set of ubuntu